docs/api.md

```diff
@@ -111,11 +111,11 @@ The response will be a JSON object containing the following keys:
   [Interaction Pings](#interaction-pings), below. This field may be null, in
   which case no click ping is required for this suggestion provider.
 
-- `provider` - A string that identifies the source of this suggestion. This can
-  be used along with `block_id` to uniquely identify this suggestion. It is not
-  intended to be directly displayed to the user.
+- `provider` - A string that identifies the provider of this suggestion, such as
+  "adM". This field is not intended to be directly displayed to the user.
 
-- ~~`advertiser`~~ - A deprecated alias of `provider`.
+- `advertiser` - The name of the advertiser, such as "Nike". Note that a `provider`
+  could have multiple `advertiser`s. This field will be displayed to the user.
```

Comment:
`This field will be displayed to the user`: that's up to the consumer of the API, right? Is the consumer that can decide whether or not it _will_ be displayed. Should we drop this sentence at all? It makes more sens eto live in Firefox :)

Reply:
Good point. Yeah, it's up to the consumers to determine whether or not to show it. I will drop it.